server/calendar/calendar.js

```diff
@@ -0,0 +1,61 @@
+var gcal = require('google-calendar');
+var calendarController = require('./calendar-controller');
+
+var calendar = {
+  addCalendarGig: function (user, gigParams) {
+    var accessToken = user.accessToken;
+    var calendar_id = user.calendar_id;
+    var calendarInfo = user.gigInformation
+    gcal(accessToken).gigs.quickAdd(calendar_id, gigInformation, function(err, data) {
+      if (err) {
+        callback({error:err});
+      }
+      callback({data: data});
+    });
+  },
+
+  removeCalendarGig: function (user, gigParams) {
+    var accessToken = user.accessToken;
+    var calendarId = user.calendar_id;
+    var gigId = user.gig_id;
+    gcal(accessToken).gigs.delete(calendar_id, gig_id, function(err, data) {
+      if (err) {
+        callback({error: err});
+      }
+      callback({data: data});
+    });
+  },
+
+  getAllCalendars: function (user, req, res) {
```

Comment:
this method and the method below both take in the req and res as params, is that on purpose? is there a reason why these do, but the functions above don't?